.github/workflows/pull-request.yml

```diff
@@ -57,10 +57,10 @@ jobs:
 
       - name: Check for linting errors in modified files
         if: steps.changed-files.outputs.only_changed != 'true'
-        env: 
+        env:
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project